style.css

```diff
@@ -1,468 +1,1180 @@
-    }
-    .time__desc {
-        font-size: 10px;
-        margin-left: -75px;
-        margin-top: -11px;
-    }
-    .about__speed {
-        padding-top: 0;
-        padding-left: 36px;
-    }
-    .speed__foto {
-        width: 55px;
-        height: 52px;
-    }
-    .speed__number {
-        font-size: 27px;
-        margin-left: -10px;
-        margin-top: 3px;
-    }
-    .speed__text {
-        font-size: 15px;
-        padding-left: 8px;
-        padding-top: 15px;
-    }
-    .speed__desc {
-        font-size: 10px;
-        line-height: 12px;
-        margin-left: -77px;
-        margin-top: -5px;
-    }
-    .test {
-        padding-top: 109px;
-        background: url("image/Test drive 1 (red 2).png") no-repeat;
-        margin-top: 48px;
-        margin-left: 36px;
-    }
-    .button {
-        margin-left: 179px;
-        margin-top: -117px;
-        width: 160px;
-        height: 40px;
-        background: #E51937;
-    }
-    .button__text {
-        font-size: 13px;
-        text-transform: uppercase;
-        padding-top: 10px;
-        padding-left: 42px;
-    }
-    .section-first__arrow {
-        padding-left: 172px;
-        padding-top: 30px;
-
-    }
+  body {
+    width: 100%;
+  }
+
+  .header {
+    width: 100%;
+    padding: 25px 47px 0 15px;
+  }
+
+  .header__burger img {
+    width: 36px;
+    height: 22px;
+  }
+
+  .section-first {
+    width: 100%;
+    background-repeat: no-repeat;
+    background-position: -5px 181px;
+    background-size: 822px;
+  }
+
+  .section-first__title {
+    padding-top: 80px;
+  }
+
+  .content {
+    width: 100%;
+    padding-left: 0;
+    margin-top: -24px;
+  }
+
+  .content__img {
+    width: 95%;
+    margin-top: 88px;
+    padding-right: 16px;
+    padding-left: 13px;
+  }
+
+  .information {
+    flex-direction: column;
+    max-width: 500px;
+  }
+
+  .information__group {
+    width: 100%;
+    margin: 0 auto;
+  }
+
+  .time__foto {
+    width: 77px;
+    height: 111px;
+  }
+
+  .information__number-title {
+    font-size: 50px;
+    padding-top: 12px;
+  }
+
+  .information__number-subtitle {
+    padding-top: 22px;
+    padding-left: 15px;
+  }
+
+  .information__number-text {
+    padding-left: 14px;
+    margin-top: -1px;
+  }
+
+  .speed__foto {
+    width: 90px;
+    height: 104px;
+    padding-top: 20px;
+    padding-left: 32px;
+    margin-bottom: 40px;
+  }
+
+  .information-block {
+    width: 100%;
+    margin: 0 auto;
+  }
+
+  .information__block__icon {
+    width: 238px;
+    background: url("image/Test drive 1 (red).png") no-repeat;
+    background-position: center;
+  }
+
+  .test__foto {
+    display: none;
+  }
+
+  .information__block__button {
+    width: 220px;
+    height: 55px;
+    background: #e51937;
+    margin-right: 40px;
+    margin-top: 7px;
+  }
+
+  .button__text {
+    font-size: 16px;
+    text-transform: uppercase;
+    padding-left: 65px;
+    color: #fff;
+  }
+
+  .section-first__arrow {
+    padding-top: 60px;
+  }
+ 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Project_AnastassiaVarabei_Evgen/pullRequest?prid=6131199)